lib/pbench/test/unit/server/conftest.py

```diff
@@ -83,8 +83,9 @@ def client(pytestconfig, monkeypatch):
 
     from pbench.server.api import create_app, get_server_config
```

Comment:
Shouldn't this be at the top of the file with all the other imports?

Reply:
ahh yeah